mkosi.presets/system/mkosi.postinst

```diff
@@ -87,3 +87,12 @@ if [ "$ID" = "centos" ] && [ "$VERSION" = "8" ]; then
     alternatives --install /usr/bin/python3 python3 /usr/bin/python3.9 1
     alternatives --set python3 /usr/bin/python3.9
 fi
+
+mkdir -p /usr/local/lib/sysusers.d
+cat >/usr/local/lib/sysusers.d/testuser.conf <<EOF
+u	testuser	4711	"Test User"	/home/testuser
+EOF
+mkdir -p /usr/local/lib/tmpfiles.d
+cat >/usr/local/lib/tmpfiles.d/testuser.conf <<EOF
+q	/home/testuser	0700	4711	4711
+EOF
```

Comment:
Probably better to just use `useradd` to create this user instead of sysusers until sysusers properly supports creating non-system users.

Reply:
As discussed elsewhere, running `useradd` in mkosi build scripts doesn't survive populating the rootfs on first-boot.